imgix/web-tools
# [PE-3726] build: use eslint

# Description
- Use ESLint for instead of JSCS and JSHint.
- Fix Node compact issue with fs promises.

# Steps to test
1. Clone this branch
2. Use a node version you'd like to test this version of web-tools in, `nvm use 14.0.0`
3. Run `npm link`
4. Open a terminal session in project you'd like to test this version in
5. Make sure the node version matches the `web-tools/` node version you linked from, `node --version` -> `14.0.0`
6. `npm link web-tools`
7. Run `npx gulp`

# Additional Info

Before/as this gets merged, all projects that currently lint with web-tools should have a lint-PR so that running web-tools in the future doesn't yield hundreds of indentation warnings (the one major change between our previous config and this one).
